influxdb2_client/src/api/label.rs

```diff
@@ -0,0 +1,294 @@
+//! Labels
+
+use crate::models::{LabelCreateRequest, LabelResponse, LabelUpdate, LabelsResponse};
+use crate::{Client, Http, RequestError, ReqwestProcessing, Serializing};
+use reqwest::{Method, StatusCode};
+use snafu::ResultExt;
+use std::collections::HashMap;
+
+impl Client {
+    /// List all Labels
+    pub async fn labels(&self) -> Result<LabelsResponse, RequestError> {
+        Ok(self.get_labels("").await?)
+    }
+
+    /// List all Labels by organization ID
+    pub async fn labels_by_org(&self, org_id: &str) -> Result<LabelsResponse, RequestError> {
+        Ok(self.get_labels(org_id).await?)
+    }
+
+    async fn get_labels(&self, org_id: &str) -> Result<LabelsResponse, RequestError> {
```

Comment:
Actually I think this function should have this signature:

```suggestion
    async fn get_labels(&self, org_id: Option<&str>) -> Result<LabelsResponse, RequestError> {
```

and then `labels` should call `self.get_labels(None)` and `labels_by_org` should call `self.get_labels(Some(org_id))`. Then the `query` should only be added to the request if needed, rather than potentially sending `orgID=`.

Reply:
Sure, I'll do that!